terraform/main.tf

```diff
@@ -18,12 +17,12 @@ module "mwaa-env" {
 #   source = "./terraform-modules/efs"
 # }
 
-module "ecs" {
-  source = "./terraform-modules/ecs"
+# module "ecs" {
```

Comment:
Whenever something is commented-out, I like to see a one-line explanation as to why. It causes less worry in the future!